File: Day 3/1470.c
int* shuffle(int* nums, int numsSize, int n, int* returnSize){
    int *ans = malloc(sizeof(int)*numsSize);
    int i=0,j=n,k=0;
    while(1){
        if(i%2==0){
            ans[i]=nums[k];
            k++;
        }
        else{
            ans[i]=nums[j];
            j++;
        }
        i++;
        if(i==numsSize){
            break;
        }
    }
    *returnSize = numsSize;
    return ans;
}

File: Day 03/1470.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int i,count=0,a=0,b=n;
        int[] ans=new int[2*n];
        for(i=0;i<nums.length;i++){
            if(i%2!=0)
              ans[count]=nums[b++];
            else
              ans[count]=nums[a++];
            count++;
        }
        return ans;
    }
}